hugo/content/downloads.md

```diff
@@ -103,6 +105,8 @@ A Flatpak Package is published and maintained by the third party Flatpak communi
 {{< content-layout/download name="iOS" href="ios" osclass="ios">}}
 {{< /content-layout/downloads >}}
 
+**Note:** The iOS App is outdated; last update in 2017.
```

Comment:
```suggestion
**Note:** The iOS  app is unmaintained; last update in 2017.
```

Reply:
What about `The iOS app is unmaintained and outdated`?

Reply:
I don't think `outdated` is accurate, unless there are issues in recent iOS versions.

Reply:
It is so old, I would assume that there are multiple issues, including non implemented security patches.

Reply:
“outdated” would imply that we know that it is outdated. “Unmaintained” says we do not know, which seems more accurate.

Reply:
Noteworthily the protocol did not change, so it should still be usable just fine - from our side.

Reply:
Well, I get your point; ok I guess the mention of the last update from 2017 should be a clear enough note for users.

Then I will add davides suggestion.